src/components/PostList/PostCard/PostCard.js

```diff
@@ -34,7 +34,10 @@ class PostCard extends React.Component {
             </Link>
           )}
         </div>
-        <div className={styles.Post__text}>
+        <div
+          className={`${styles.Post__text} ${!thumbExists &&
+            styles.Post__noThumbnail}`}
```

Comment:
Should be:

```
          className={`${styles.Post__text} ${thumbExists ? '' :
            styles.Post__noThumbnail}`}

```

Otherwise you pass the string`'undefined'` as a class name

Reply:
Ah, good catch.